cli/src/commands/init.rs

```diff
@@ -158,35 +165,59 @@ fn prompt_lockfile_name() -> io::Result<String> {
         .flat_map(|entry| entry.file_name().to_str().map(str::to_owned))
         .collect();
 
-    // Prompt if a lockfile was found.
+    // Prompt for selection any lockfile was found.
```

Comment:
Wording could be improved, for example:
```suggestion
    // Prompt for selection of any lockfile that was found.
```
or:
```suggestion
    // Prompt the user to pick one or many among the lockfiles that were found.
```

Reply:
Was just missing an `if`.